addons/account/models/account_move_send.py

```diff
@@ -57,7 +57,7 @@ def get_setting(key, from_cron=False, default_value=None):
             return custom_settings.get(key) if key in custom_settings else move.sending_data.get(key) if from_cron else default_value
 
         vals = {
-            'sending_methods': get_setting('sending_methods', default_value={self._get_default_sending_method(move)}) or {},
+            'sending_methods': get_setting('sending_methods', default_value=[self._get_default_sending_method(move)]) or [],
```

Comment:
Hello, 
I agree with the wrong return type above but why this change ?

Reply:
Helllo @clbr-odoo Because `sending_methods` here is expected to be a list. See:

https://github.com/odoo/odoo/blob/18.0/addons/account/models/account_move_send.py#L681
https://github.com/odoo/odoo/blob/18.0/addons/account/wizard/account_move_send_wizard.py#L181
https://github.com/odoo/odoo/blob/18.0/addons/account/wizard/account_move_send_wizard.py#L236
https://github.com/odoo/odoo/blob/18.0/addons/account/wizard/account_move_send_wizard.py#L245
https://github.com/odoo/odoo/blob/18.0/addons/account/wizard/account_move_send_wizard.py#L271
https://github.com/odoo/odoo/blob/18.0/addons/account/tests/test_account_move_send.py#L165
https://github.com/odoo/odoo/blob/18.0/addons/account/tests/test_account_move_send.py#L165